GirafIntegrationTest/test.py

```diff
@@ -1,13 +1,31 @@
 import unittest
+import sys
 from requests import get
 from requests.exceptions import ConnectionError
+from argparse import ArgumentParser
 from testlib import GIRAFTestResults, GIRAFTestRunner, compare, BASE_URL
 
+parser = ArgumentParser()
+parser.add_argument('--missing', action='store_true', help='print untested endpoints')
+parser.add_argument('--tested', action='store_true', help='print tested endpoints')
+
+args = parser.parse_args()
+
+# importing util globally borks the tests due to test class imports
+if args.missing:
+    from util import missing_endpoints
+    missing_endpoints()
+    sys.exit(0)
+if args.tested:
+    from util import tested_endpoints
+    tested_endpoints()
+    sys.exit(0)
+
 try:
     result = get(f'{BASE_URL}v1/Error').json()
```

Comment:
Running the test command you mentioned returns the error seen on the image.

![image](https://user-images.githubusercontent.com/32843490/79799052-25a3ab80-835a-11ea-9d02-5e1e2bf62b39.png)